### ISSUE TITLE
"USB Composite Device" with failed "This device cannot start. (Code 10)"

### ISSUE THREAD

**Issue by uskr** [brandonlw/Psychson#19]
I successfully configured the required environment and got the correct Burner .bin and the supported  Supersonic Patriot USB.

After writing the changed firmware, when I connect the USB to Windows, it shows a "USB Composite Device" with an exclamation mark and "This device cannot start. (Code 10)".

I saw some reports of people saying that although a USB 3.0 stick is required, it only works on a USB 2.0 port after the custom firmware is written on the USB Stick. I personally don't have any USB 2.0 port on any of my machines to test. So I just wanted to confirm if anyone is having that problem too.

I can still write new firmwares or the original firmware to the stick, by forcing windows to use a mass storage device driver on that USB device, so it assigns a letter to it (although unusable in the sense of a mass storage device).

In short, is that working as intended (to only work on usb 2.0 ports after flashed)? Anyone experiencing the same/similar problem?

**Comment by uskr** [comment]
On a side note, after having a completelly dead USB (no light blinking on connecting) I was able to shortcut the 2 pins to force boot mode and re-flash the original firmware.

However, I could only successfully write the original firmware back and resuscitate the USB back to normal using the BN03V114M.bin burner. All others showed all kind of different behaviors.

But when I write the custom hid.bin firmware I get the following behaviors:
- With BN03V104M.bin burner: "USB Composite Device" with failed "This device cannot start. (Code 10)"
- With BN03V114M.bin burner: "USB Mass Storage Device" with failed "This device cannot start. (Code 10)" and message "The USB device returned an invalid USB configuration descriptor."

On the bright side, it doesn't matter how dead the USB become during the tests (no blinking lights at all when connected, even), I can always flash back back the original firmware using the BN03V114M.bin burner (all others fail at different stages) by short circuiting the 2 pins manually to force the boot mode.

Will test flashing the hid.bin file with other burners now to see if I get anything different.

**Comment by uskr** [comment]
Just a follow up, and this issue can be closed, as far as I am concerned.

All the steps on the previous two posts were happening while using the USB3 Patriot Stick on a USB3 port. On different machines, to make sure it was not a problem with the host itself.

Today I could test everything again. Using the same USB3 Patriot Stick, but on a USB2 port. And everything worked first try. Could flash and re-flash whetever I wanted and all got properly reconized by the OS without a problem.

In short, use the USB3 Patrio stick (2303), but on a USB2 port.
